### PR TITLE
assembly guide: added other resources

### PR DESCRIPTION
**Pull request by AWSjswinney**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by AWSNB — APPROVED** [review]
LGTM,   maybe we add a comment that in general, while these are great reocmmendations, we recommend testing and experimenting on the actual EC2 instnace

File: arm64-assembly-optimization.md [hunk 1]
```diff
@@ -24,6 +24,34 @@ Some techniques for writing optimized assembly:
 
 We will be adding more sections to this document soon, so check back!
 
+### Other resources
+
+This guide aims only to provide tips on optimizing assembly or intrinsics code.
+There are other resources available on the web which can help you to learn
+concepts or refer to as references. Here a list of helpful pages.
```
Comment (wash-amzn) [inline arm64-assembly-optimization.md:31]: I think this is a little off: "There are other resources available on the web which can help you to learn
concepts or refer to as references."

There are other resources available on the web which can
- help you to learn concepts
- refer to as references.

There are other resources available on the web which can help you to
- learn concepts
- refer to as references

The "refer to as references" branch of both parsings don't come out right
- "There are other resources available on the web which can refer to as references"
- "There are other resources available on the web which can help you to refer to as references"

I hope that makes sense

Comment (AWSjswinney) [reply arm64-assembly-optimization.md]: Thanks for commenting on that. I made a change to the wording. Does it look better to you?

**Comment by AWSjswinney** [comment]
> LGTM, maybe we add a comment that in general, while these are great reocmmendations, we recommend testing and experimenting on the actual EC2 instnace

Thanks for taking a look. I added a couple of sentences about testing on Graviton itself.

File: arm64-assembly-optimization.md (hunk 1 above)
Comment (wash-amzn) [reply arm64-assembly-optimization.md]: I'd say it's still a little awkward, but it does parse now.

**Review by wash-amzn — APPROVED** [review]

**Pull request merged**